about.html

```diff
@@ -0,0 +1,172 @@
+              </li>
+
+              <li>
+                <a href="#news" class="nav-links news-link">News</a>
+              </li>
+
+              <li>
+                <a href="#" class="nav-links active">AiT Summit</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+
+      <section class="landing-about">
+        <div class="landing-text">
+          <h1>TECH IN AFRICA GLOBAL SUMMIT 2022</h1>
+          <p>
+            Tech in Africa Summit connects tech leaders from the African
+            ecosystem and international players under one roof. Network with key
+            stakeholders, including tech corporates, mobile operators, fintechs,
+            DeFi & crypto ventures, investors, leading start-ups, regulators and
+            industry stakeholders driving business and investment forward.
+          </p>
+          <span class="contact-mail">
+            Please contact us per Email for any further questions about AiT
+            summit 2022!
+            <a href="#"><span>aitsummit2022@techafrica.com</span></a>
+          </span>
+        </div>
+      </section>
+
+      <section class="gallery">
+        <h2>
+          Tech in Africa <br />
+          Global Summit 2022
+        </h2>
+        <p>
+          The logo of AiT Global summit 2015 was designed by me as one of the
+          technical decision that I made to personalise my capstone project.
+        </p>
+
+        <div class="logo-show">
+          <img src="images/spiral_logo.png" alt=" Summit Logo" />
+        </div>
+        <div class="line"></div>
+
+        <div class="photo">
+          <h3>See the past AiT Global summit</h3>
+          <p>
+            Take a loot at the last two AiT summits which took place in Lagos
+            and Capetown
+          </p>
+
+          <div class="pastPhotos">
+            <img src="images/pic1.png" alt="Photo-gallery" />
+            <img src="images/pic2.png" alt="Photo-gallery" />
+          </div>
+        </div>
+      </section>
+
+      <section class="partners">
+        <h2>Partners</h2>
+
+        <hr />
+
+        <div class="partner1">
+          <a href="#"
+            ><img src="images/microverse.png" alt="Partner1" />Microverse</a
+          >
+
+          <a href="#"> <img src="images/google.jpg" alt="Partner1" />Google</a>
+
+          <a href="#">
+            <img src="images/mozilla.jpg" alt="Partner1" />mozilla</a
+          >
+        </div>
+
+        <div class="partner2">
+          <a href="#">
+            <img src="images/icon-placeholder.jpg" alt="Partner1" />Partner</a
+          >
+
+          <a href="#">
+            <img src="images/icon-placeholder1.jpg" alt="Partner1" />Partner</a
+          >
+        </div>
+      </section>
+      <footer>
+        <div class="copy-right">
+          <img src="images/footer_logo.png" alt="Footer" />
+          2022 Tech in Africa summit Abuja. Some Rights Reserved.
+        </div>
+      </footer>
```

Comment:
- [ ] Please ensure that the footer in the about page matches the one in the home page with only the background color being different

### About Page
![image](https://user-images.githubusercontent.com/43995830/163595903-314bf752-d5bf-4945-87c2-c08c8a2c7c93.png)

### Home Page
![image](https://user-images.githubusercontent.com/43995830/163595986-a19a2ab0-93b5-491f-b6f2-2d0630a24990.png)